lib/http-proxy.js

```diff
@@ -1,7 +1,7 @@
 var http      = require('http'),
     https     = require('https'),
     url       = require('url'),
-    httpProxy = require('./http-proxy/');
+    httpProxy = require('./http-proxy/index.js');
```

Comment:
This also breaks other packages depending on [node-resolve](https://github.com/substack/node-resolve/issues/51) (including Jest).

I'd be happy to submit another PR isolated to just this change. Let me know.